pick-infrastructure/src/main/kotlin/com/pickdsm/pickserverspring/global/entity/BaseUUIDEntity.kt

```diff
@@ -0,0 +1,17 @@
+package com.pickdsm.pickserverspring.global.entity
+
+import org.hibernate.annotations.GenericGenerator
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.MappedSuperclass
+
+@MappedSuperclass
+abstract class BaseUUIDEntity (
+    @Id
+    @GeneratedValue(generator = "uuid2")
+    @GenericGenerator(name = "uuid2", strategy = "uuid2")
+    @Column(columnDefinition = "BINARY(16)")
+    val id: UUID = UUID(0,0)
+)
```

Comment:
Main 브런치 풀 받아서 UUID version 7 적용해주세요

```suggestion
@MappedSuperclass
abstract class BaseUUIDEntity (
    @Id
    @Column(columnDefinition = "BINARY(16)", nullable = false)
    val id: UUID = UuidCreator.getTimeOrderedEpoch()
)
```